### PR TITLE
fix: equal height issue solved

### PR DESCRIPTION
**Pull request by nishtha53**
image

Solved equal height issue. Please let me know if any changes required.

### CODE REVIEW

**Comment by ankitwww** [comment]
Hey @nishtha53 - great job! Could you please remove all other files except components/HacktoberedCard.tsx from PR... they don't seem to be related to the PR

File: .env.local.example [hunk 1]
```diff
@@ -1,7 +1,7 @@
 NEXTAUTH_URL=http://localhost:3000
-NEXTAUTH_SECRET= # Linux: `openssl rand -hex 32` or go to https://generate-secret.now.sh/32
+NEXTAUTH_SECRET= f1a4e36f399c85239978c9a987002674
```
Comment (ankitwww) [inline .env.local.example:2]: Please remove this file from PR

**Comment by nishtha53** [comment]
Removed but still checks are not successful.

**Comment by ankitwww** [comment]
@nishtha53 don't worry abt vercel runs - please fix the files first. Your PR still have other files... please drop them from PR (or raise a new PR with only one file components/HacktoberedCard.tsx)

**Comment by nishtha53** [comment]
@ankitwww now all other files are removed from PR except components/HacktoberedCard.tsx

**Comment by ankitwww** [comment]
> @ankitwww now all other files are removed from PR except components/HacktoberedCard.tsx

Latest commit is correct, but the PR still have prev commits. Please drop prev commits (or raise a new PR)

**Comment by nishtha53** [comment]
@ankitwww In new PR all commits are also coming so trying to do in same one

**Pull request closed without merging**